packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -357,7 +357,8 @@ class WebpackCLI {
 
                     try {
                         pkg = await promptInstallation(pkg, () => {
-                            logger.error(`For using this command you need to install: '${green(commandName)}' package`);
+                            const pkgName = pkg.slice(pkg.indexOf('/') + 1);
+                            logger.error(`For using this command you need to install: '${green(pkgName)}' package`);
```

Comment:
```suggestion
                            logger.error(`For using this command you need to install: '${green(pkg)}' package`);
```

Reply:
In fact, the subsequent prompt displays the package name along with the `@webpack-cli` prefix.

<img width="1318" alt="Screenshot 2021-01-10 at 12 24 44 PM" src="https://user-images.githubusercontent.com/25279263/104116477-d9acf280-533e-11eb-9982-b42cfd2dd217.png">